src/compiler/commandLineParser.ts

```diff
@@ -244,6 +251,13 @@ export const libs = libEntries.map(entry => entry[0]);
  */
 export const libMap = new Map(libEntries);
 
+/**
+ * Maps a lib name to its filename in the TypeScript package.
+ */
+export function getLibFileName(libName: string): string | undefined {
+    return libMap.get(libName);

```

Comment:
Should this lowercase `libName` eg. `ES5` should also get `lib.es5.d.ts`.. Eg. look at `getLibFileNameFromLibReference`

Reply:
I think my intent here was that you only ever give this function strings you get from the other APIs, though I just haven't had time to deal with this PR (besides updating it post-dprint) so I can't remember anymore 😅